gui_qt/keyword_workflow.py

```
@@ -107,14 +107,14 @@ def _finish_build(self, output: str) -> WorkflowUpdate:
                     status="done",
                     heading="没有可提取的关键词源行",
-                    message="当前项目没有可用于关键词提取的 TL 文本行。",
+                    message="当前项目没有可用于关键词提取的翻译文本。",
                     facts=[],
                 )
             self._pending_steps.clear()
             return WorkflowUpdate(
                 status="failed",
                 heading="无法定位关键词请求包",
-                message="build-keywords 已结束，但输出中没有请求包路径；请查看原始输出。",
+                message="关键词任务准备未完成，请查看诊断日志。",
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)